SensorsAnalyticsSDK/src/main/java/com/sensorsdata/analytics/javasdk/SensorsAnalytics.java

```diff
@@ -156,12 +160,18 @@ public BatchConsumer(final String serverUrl, final int bulkSize) {
     }
 
     public BatchConsumer(final String serverUrl, final int bulkSize, final boolean throwException) {
-      this(serverUrl, bulkSize, 0, throwException);
+      this(serverUrl, bulkSize, 0, throwException, null);
```

Comment:
I don't think we should be passing null here

Reply:
Good catch